react/features/base/color-scheme/defaultScheme.js

```diff
@@ -41,6 +41,7 @@ export default {
         button: 'rgb(255, 255, 255)',
         buttonToggled: 'rgb(38, 58, 76)',
         buttonToggledBorder: getRGBAFormat('#a4b8d1', 0.6),
-        hangup: 'rgb(227,79,86)'
+        hangup: 'rgb(227,79,86)',
+        hangupMenu: 'rgb(0, 119, 225)'
```

Comment:
Please use a token instead. This file will disappear soon. See base/ui/

Reply:
Has become obsolete and has been removed.